roles/branch-validation/tasks/main.yml

```diff
@@ -68,17 +61,14 @@
     nics:
       - net-name: internal
     name: "validate-{{ item }}"
+    availability_zone: "nova:{{ item }}"
```

Comment:
This still works on 2.1 stacks, and if we don't schedule using hints there's no guarantee that hosts get an instance.